### ISSUE TITLE
OL error when zooming in: layer.unrender is not a function

### ISSUE THREAD

**Issue by Siedlerchr** [indoorequal/openlayers-indoorequal#11]
Hi,

thanks for the cool plugin! I tried to integrate it in my app and so far it shows the heatmap. However, when I try to zoom further in, I get an error from OpenLayers. This is definitely related to the plugin.
I followed the example and can see the blue heatmap stuff, but the concrete building is not shown
Any idea what this could be?

I'm using the latest version of "ol": "^6.14.1", inside Angular 12 if that helps.

```
ERROR TypeError: layer.unrender is not a function
    renderFrame Composite.js:118
    renderFrame_ PluggableMap.js:1291
    animationDelay_ PluggableMap.js:245
    render PluggableMap.js:1197
    handleViewPropertyChanged_ PluggableMap.js:1120
    dispatchEvent Target.js:118
    notify Object.js:185
    set Object.js:218
    applyTargetState_ View.js:1498
    adjustResolutionInternal View.js:1386
    adjustResolution View.js:1370
    adjustZoom View.js:1396
    handleEvent MouseWheelZoom.js:215
    handleMapBrowserEvent PluggableMap.js:984
    handleBrowserEvent PluggableMap.js:941
    sentryWrapped helpers.js:74
    Angular 15
    _wrapEventTarget trycatch.js:185
    instrumentDOM instrument.js:458
    handleTargetChanged_ PluggableMap.js:1095
    dispatchEvent Target.js:118
    notify Object.js:181
    set Object.js:218
    setProperties Object.js:231
    PluggableMap PluggableMap.js:385
    Map Map.js:86
    initMapAndLayerData map.component.ts:88
    ngOnInit map.component.ts:68
    sentryWrapped helpers.js:74
    Angular 17
    _wrapTimeFunction trycatch.js:94
    ngOnInit map.component.ts:67
    Angular 19
instrument.js:109
```

**Comment by Siedlerchr** [comment]
@francois2metz Indeed, Ol 6.12 seems to work!

**Comment by francois2metz** [comment]
Hi @Siedlerchr. I believe the issue is now fixed in the v0.0.7.

**Comment by Siedlerchr** [comment]
@francois2metz  Thanks for the quick fix! Works fine now! 👍
